.github/workflows/rust.yml

```diff
@@ -22,3 +22,5 @@ jobs:
       run: cargo build --verbose
     - name: Run tests
       run: cargo test --verbose
+    - name: Check formatting
+      run: cargo fmt --check
```

Comment:
Here's what I've been doing in my crates:

```
      - name: Fmt
        run: cargo fmt --all -- --check
```

Haven't checked what the difference is

Reply:
From `cargo fmt --help`:  
```
--all
Format all packages, and also their local path-based dependencies
```

Reply:
So i guess it will probably only be relevant if we want to add derive suport